### ISSUE TITLE
modifier for positions (top, left ...)

### ISSUE THREAD

**Issue by MoOx** [cssrecipes/utils#9]
Currently we have r-Grid-cell--alignTop etc
What if we handle this here with r-*--align* ?
Top,Middle,Bottom,Left,Center,Right ?

**Comment by bloodyowl** [comment]
agreed for vertical alignment, makes sense when `r-*-XofY` are set. not sure about left, center & right though
